src/pdmat.jl

```diff
@@ -65,8 +65,8 @@ end
 
 ### quadratic forms
 
-quad(a::PDMat, x::StridedVector) = dot(x, a * x)
-invquad(a::PDMat, x::StridedVector) = dot(x, a \ x)
+quad(a::PDMat, x::AbstractVector) = dot(x, a * x)
```

Comment:
I'd like to have @andreasnoack weigh in, but that seems reasonable. It would let people know that `*` and `\` are the minimal interface they'd need to define for everything else to work.

In this case, you'd also want to widen `quad!` and `invquad!` everywhere, I think.